src/tools/wallmap/wallmap.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"math"
+	"os"
+
+	"github.com/fogleman/gg"
+	"golang.org/x/image/bmp"
```

Comment:
We would have to sync with @duplo (@duploduplo ?!) for this as the
resulting image is also read client-side, in order to ease mouse picking,
eg show a red tower instead of green when the player is trying to build a
turret on a wall.

But it's true that a PNG being lossless and compressed, it would be better
to use PNG instead of BMP, thus  substantially decreasing the assets size,
and allowing to discard an external depencency on the Go side. Good catch 😉

Le 24 août 2016 01:30, "Giovanni Bajo" <notifications@github.com> a écrit :

> In src/tools/wallmap/wallmap.go
> <https://github.com/RookieGameDevs/surviveler/pull/3#discussion_r75970851>
> :
>
> > @@ -0,0 +1,95 @@
> > +package main
> > +
> > +import (
> > +	"flag"
> > +	"fmt"
> > +	"math"
> > +	"os"
> > +
> > +	"github.com/fogleman/gg"
> > +	"golang.org/x/image/bmp"
>
> any reason not to use image/png instead? It's part of the standard library
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/RookieGameDevs/surviveler/pull/3/files/1f37e8cab422f8d46587682b13db9c63d44b6718#r75970851>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAdF6t0ytm1W8rCaUXXhzZZxUcRoR5eQks5qi4J9gaJpZM4JrcmN>
> .
>